src/Teams/Services/ManageTeamsService.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Linq;
+using Teams.Data;
+using Teams.Models;
+using Teams.Repository;
+using Teams.Security;
+
+namespace Teams.Services
+{
+    public class ManageTeamsService : IManageTeamsService
+    {
+        private readonly ICurrentUser _currentUser;
+        private IRepository<Team, int>  _teamRepository;
+        private IRepository<TeamMember, int> _teamMemberRepository;
+        public ManageTeamsService(ICurrentUser currentUser, IRepository<Team, int> teamRepository, IRepository<TeamMember, int> teamMemberRepository)
+        { 
+            _currentUser = currentUser;
+            _teamRepository = teamRepository;
+            _teamMemberRepository = teamMemberRepository;
+        }
+        public List<Team> GetMyTeams()
+        {
+            List<Team> myteams = new List<Team>();
+            myteams.AddRange(_teamRepository.GetAll().ToList<Team>().Where(team => team.TeamOwner == _currentUser.Current.Id()));
+            myteams.AddRange(GetMyMemberTeams());
```

Comment:
Please do it just in one Where 

https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.entityframeworkqueryableextensions.include?view=efcore-3.1